knowledge-base/chart-draw-horizontal-vertical-lines.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Charts - Draw Horizontal and Vertical Lines
+description: How to draw horizontal lines in Charts. How to draw vertical lines in Charts.
```

Comment:
Don't repeat the same question in the description. Use the available space to add information about the implementation, for example.